experimental/generation/generator/packages/library/templates/standard/date-timeProperty.lg.lg

```diff
@@ -0,0 +1,27 @@
+[generator.lg](../generator.lg)
+
+# filename
+- ${propertyFilename()}
+
+# template
+- ```
+${importLG('library')}
+${importLG('library-Missing')}
+${AskTemplate()}
+${nameTemplate()}
+${ValueTemplate()}
+```
+
+# AskTemplate
+- ```
+> Template for prompting the user for the '${property}' property.
+${'#'} ${property}_getPromptText
+- \${getStringPromptText('${property}')}
+```
+
+# ValueTemplate
+-```
+> Template for generating display text for the value of the '${property}' property.
+${'#'} ${property}_getValueText(val)
+- \${getValueText(val)}
```

Comment:
Are these different than the normal ones?

Reply:
This is the same as ObjectProperty.lg.lg